src/components/map/MapService.js

```diff
@@ -935,15 +940,19 @@ goog.require('ga_urlutils_service');
           return $http.get(url).then(function(response) {
             // Live modifications for 3d test
             if (response.data) {
+              // pngjpeg is not supported by zeitreihen in 3D
               var ids = [
-                'ch.swisstopo.swissimage-product',
-                'ch.swisstopo.pixelkarte-farbe',
-                'ch.swisstopo.pixelkarte-grau',
                 'ch.swisstopo.zeitreihen'
               ];
               angular.forEach(ids, function(id) {
                 if (response.data[id]) {
-                  response.data[id].config3d = id + '_3d';
+                  var config3dId = id + '_3d';
+                  var config3d = angular.extend({},
+                      response.data[id], config3d);
+                  config3d.format = 'jpeg';
+                  config3d.useMapProxy = true;
+                  response.data[config3dId] = config3d;
+                  response.data[id].config3d = config3dId;
```

Comment:
It because it's not a native layers and is therefore not stored in BD (as a result not in the service). We do all that just to change the format form `pngjpeg` to `jpeg`.